"""
Задание 2.

Создайте собственный!!! класс-исключение, обрабатывающий ситуацию деления на нуль.
Проверьте его работу на данных, вводимых пользователем. При вводе пользователем нуля
в качестве делителя программа должна корректно обработать эту ситуацию и не завершиться с ошибкой.
"""


class OwnError(Exception):
    pass


inp_data = float(input("Введите число1: "))
inp_data2 = float(input("Введите число2: "))

try:
    if inp_data2 == 0:
        raise OwnError("Нельзя делить на ноль")
except OwnError as err:
    print(err)
